string serial                         # Lighthouse serial number
---
deepdive_ros/Lighthouse[] lighthouses     # Motor calibration info
